pom.xml

```diff
@@ -56,33 +59,10 @@
   </build>
 
   <dependencies>
-    <dependency>
```

Comment:
Github-api shades internal dependencies such as jackson.  No longer need to sync with other plugins.